README.md

```diff
@@ -77,24 +77,28 @@ Fetch a static file belonging to a preset with id `id`.
 
 ### Media
 
-#### `PUT /media?file=PATH`
+#### `PUT /media?file=PATH&thumbnail=PATH
 
 Save a piece of media (photos only), identified by the absolute file path `PATH`
 to the database. `PATH` should be URL encoded.
 
-A single JSON object is returned, with, at minimum, the `id` field set, to
-uniquely identify the uploaded media:
+Optionally, a thumbnail file path may also be included.
+
+A single JSON object is returned, with the base `id`, the original image's ID, and, if a thumbnail as given, the thumbnail ID:
 
 ```json
 {
-  "id": "225961fb85d82312e8c0ed511",
-  "type": "image/jpg"
+  "id": "225961fb85d82312e8c0ed511.jpg",
+  "original": "original/225961fb85d82312e8c0ed511.jpg",
```

Comment:
Do we need these original & thumbnail properties? What info do they give us?
I'm thinking maybe to drop them.

Reply:
I wasn't sure which route I wanted to take:

1. treat `original` and `thumbnail` as opaque IDs for the client to use (allows future change easily)
2. expose a single `id` and let the client formulate URLs

Reply:
OK, let's just expose `id`. Done.